qbnb_test/register_injection_test/resgister_injection_test.py

```diff
@@ -0,0 +1,63 @@
+"""
+NoSql injection test for register function. The injection payload
+used for testing is not the file provided in the assignment, because
+our project uses non relational database instead of sql.
+"""
+from pathlib import Path
+from qbnb.models import User, user_register
+from qbnb import db
+current_folder = Path(__file__).parent
+
+
+def test_register_injection_email():
+    """
+    test case for injection in register function
+    tested parameter: email
+    """
+    input_file = open(current_folder.joinpath(
+        "input.txt"))
+    for text in input_file:
+        try:
+            user_register(text, "A123456a!", "test0user")
+        except Exception as e:
+            assert False
+    user = User.objects()
+    for i in user:
+        i.delete()
+    input_file.close()
+
+
+def test_register_injection_password():
+    """
+    test case for injection in register function
+    tested parameter: password
+    """
+    input_file = open(current_folder.joinpath(
+        "input.txt"))
+    for text in input_file:
+        try:
+            user_register('test@test.com', text, "test0user")
+        except Exception as e:
+            assert False
+    user = User.objects()
+    for i in user:
```

Comment:
This would delete all users in the collection if I'm not wrong, best to only delete the users which your test case has created